package.json

```diff
@@ -38,7 +38,7 @@
     "csurf": "1.10.0",
     "errorhandler": "1.5.1",
     "express": "4.17.1",
-    "express-fileupload": "^0.4.0",
+    "express-fileupload": "^1.3.1",
```

Comment:
it probably makes sense to have this in it's own PR too. I like how the other version bumps are in a commit named "bump x from y to z" with a "dependencies" tag so it is easy to spot these in the git history and to update history.md. 

Is #592 the only dependabot PR for fileupload? It seems to be updating more than just fileupload.